app/javascript/components/applications/application_form/PrimaryApplicantSection.js

```diff
@@ -16,35 +16,10 @@ let mailingAddressFieldMap = {
   zip: 'mailingZip',
 }
 
-const validates = (fun, message) => (value) => {
-  return fun(value) ? null : message
-}
-
-const isOldEnough = (dateOfBirth) => {
-  const years = moment().diff(dateOfBirth,'years')
-  return years >= 18
-}
-
-const isValidDate = (dateOfBirth) => {
```

Comment:
looks like this one didn't make it over to your validation page.

Reply:
fixed